src/RedisCache/RedisCache/help/Remove-AzRedisPrivateEndpointConnection.md

```diff
@@ -0,0 +1,183 @@
+﻿
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.RedisCache.dll-Help.xml
+Module Name: Az.RedisCache
+online version: https://docs.microsoft.com/powershell/module/az.rediscache/remove-azredisprivateendpointconnection
+schema: 2.0.0
+---
+
+# Remove-AzRedisPrivateEndpointConnection
+
+## SYNOPSIS
+Remove a Private Endpoint Connection from a Redis Cache.
```

Comment:
```suggestion
Remove a private endpoint connection from Azure Cache for Redis.
```
We don't usually capitalize Private Endpoint Connection in the Azure docs when discussing it.

https://docs.microsoft.com/en-us/azure/azure-cache-for-redis/cache-private-link